ref/general/LDAP-user-registries-authentication-authorization.adoc

```diff
@@ -0,0 +1,56 @@
+:page-layout: general-reference
+:page-type: general
+= LDAP user registries for authentication and authorization
+
+You can configure your applications that run on Open Liberty to manage authentication with a Lightweight Directory Access Protocol (LDAP) user registry.
+
+LDAP is an open protocol that supports authentication and authorization services.
+LDAP specifies the language that application servers use to communicate with user registries that store security information such as usernames, passwords, and user groups.
+Although a link:/docs/ref/general/#basic-registry.html[basic user registry] might be sufficient during development and testing of an application, an LDAP user registry provides the extra security that is required for applications in production.
+
+LDAP, the industry standard for directory servers, provides a means to create, modify, search, and delete users and groups in your applications.
+There are many LDAP servers available, such as the link:https://www.ibm.com/support/knowledgecenter/SSVJJU_6.4.0/com.ibm.IBMDS.doc_6.4/c_po_SDS_overview.html[IBM Security Directory Server] and the Microsoft Active Directory.
+There are also open source LDAP servers, such as OpenLDAP and ApacheDS.
+
+== Federated LDAP registries
+
+You can combine multiple LDAP registries into a single federated registry to manage authentication and authorization.
+Federated registries provide a single logical view of multiple user registries and repositories.
+
+In cases where user and group information is spread across multiple registries, federation enables Open Liberty to use distributed user information in a unified manner, with a continuous store of information.
+Different types of registries can be federated to combine information from LDAP, basic, and custom user registries and repositories into a common registry. However, user registries that are configured with the `quickStartSecurity` element cannot be federated with other registries.
+
+LDAP registries are federated by default.
+If you configure more than one LDAP registry in your `server.xml` file, then the registries are automatically federated into a common registry.
+This default federation occurs regardless of whether the LDAP registries are separate data stores or two subtrees of the same LDAP registry.
+Similarly, if one or more LDAP registries are configured with one or more basic or custom registries in the `server.xml` file, then they are federated by default.
+
+You can adjust the configuration of federated registries by enabling the link:/docs/ref/feature/#federatedRegistry-1.0.html[Federated User Registry] feature and configuring the `federatedRepository` element.
+For more information, see link:/docs/ref/config/#federatedRepository.html[User Registry Federation].
+
+== LDAP registries and the Social Media Login feature
+
+You can use Open Liberty's Social Media Login feature to help manage authentication with a configured LDAP registry.
```

Comment:
"You can use Open Liberty's Social Media Login feature to help manage authentication with a configured LDAP registry."
to
"You can use Open Liberty Social Media Login feature to help manage authentication with a configured LDAP registry."